notebook/static/notebook/js/codecell.js

```diff
@@ -335,10 +335,10 @@ define([
         this.render();
         this.events.trigger('execute.CodeCell', {cell: this});
```

Comment:
@rgbkrk When a CodeCell is executed, `'execute.CodeCell'` is triggered, but apparently no event is triggered when execution completes.